src/utils/optionsHaveIgnoredPseudoClass.js

```diff
@@ -0,0 +1,17 @@
+import matchesStringOrRegExp from "./matchesStringOrRegExp"
```

Comment:
@jeddy3 i think no, some `optionsHave*` utils have difference logic and it would be nice to be sure they